### PR TITLE
up and down now change the gravity of the block

### PR DESCRIPTION
**Pull request by adamhundley**
now when a user clicks up and down the gravity reverses

### CODE REVIEW

File: lib/BlockParty.js [hunk 1]
```diff
@@ -1,6 +1,17 @@
 import React, { Component } from 'react';
 import PartySquare from './PartySquare';
 
+const KEY = {
+  LEFT:  37,
+  RIGHT: 39,
+  UP: 38,
+  DOWN: 40,
+  A: 65,
```
Comment (weilandia) [inline lib/BlockParty.js]: Do we still need this?

File: lib/PartySquare.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (weilandia) [inline lib/PartySquare.js]: Let's remove this.

File: lib/PartySquare.js [hunk 3]
```diff
@@ -8,6 +9,24 @@ export default class PartySquare {
...
   }
 
+  respondToUser(key){
+    if(key === 38 || key === 40){
+      this.manageGravity(key) ;
+    }
+  }
+
+  manageGravity(key){
+    if(key === 38){
```
Comment (weilandia) [inline lib/PartySquare.js]: Can we simplify this with ```if(key === 38 && this.gravity)```

Comment (adamhundley) [inline lib/PartySquare.js]: Nice catch! I'll fix this.

**Comment by adamhundley** [comment]
hey @weilandia. check it out. i fixed this issues

**Comment by weilandia** [comment]
Looks good.

**Pull request merged**